### ISSUE TITLE
Error retrieving character list, try again (Timeout was reached)

### ISSUE THREAD

**Issue by xBionickx** [Openarl/PathOfBuilding#1080]
As title suggests, cannot load character list.

Never use PoB before, have newest version (v1.4.95) see attached pic.

![path of building_2018-06-18_12-14-58](https://user-images.githubusercontent.com/40356784/41533155-bb122528-72f1-11e8-8891-86b8947a42ea.png)

**Comment by Openarl** [comment]
I'm not sure what could be causing that; is it happening every time you try to import?

**Comment by xBionickx** [comment]
Yep, every time I've tried, it does this. Restarted the client multiple times, and on various times after turning the PC on for the first time of the day etc.

**Comment by Openarl** [comment]
Are you using a proxy or similar?

**Comment by xBionickx** [comment]
Nope, straight through a modem/ethernet, no Proxy/VPN.

**Comment by Openarl** [comment]
That's odd; usually, import problems involve issues with SSL, but that doesn't seem to be the case here. I haven't got any more ideas, sorry.

**Comment by xBionickx** [comment]
That's no problem, also though at the bottom left it keeps Checking... next to the current version.

**Comment by Openarl** [comment]
Yeah, the program just doesn't seem able to connect to anything at all. You could try disabling/re-configuring your firewall; I guess that could be interfering.

**Comment by xBionickx** [comment]
I don't think I've done anything wrong.. I extracted the .zip into the HDD I use, and from there, just run the PoB Application, nothing else has to happen before, right?

Disabled the Windows Firewall and still doesn't retrieve character list.

**Comment by Openarl** [comment]
It should just work right after installing. I have no idea why that isn't the case for you.

**Comment by kjeele** [comment]
getting ssl connect error.
pob updates itself, but cant import character.
what can i do to get rid of this error?

**Comment by ImGr0k** [comment]
I encountered such a problem 2 days ago.
I have Windows 10 and drivers for the Realtek network card. There was an update of Windows, after that there was such an error. I deleted the driver on the network card and installed the latest drivers. I don’t know what is connected, but the solution helped.

**Comment by ertanturan** [comment]
Same error here ! . Can't import nothing.

**Comment by Vipera222** [comment]
Hi got simillar issue today its says (Response code. 403) in the brackets

**Comment by Vipera222** [comment]
Restarting repaired it tho
